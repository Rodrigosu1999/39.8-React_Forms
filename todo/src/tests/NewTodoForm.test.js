import React from 'react';
import { getByText, render, screen } from '@testing-library/react';
import NewTodoForm from '../components/NewTodoForm';

it('Renders App', () => {
  render(<NewTodoForm />);
});

it('Passes Snapshot', () => {
  const {asFragment} = render(<NewTodoForm />);
  expect(asFragment()).toMatchSnapshot();
});

it('renders correct text', () => {
  const {getByText} = render(<NewTodoForm />);
  console.log(getByText('Add new Todo!'))
});
