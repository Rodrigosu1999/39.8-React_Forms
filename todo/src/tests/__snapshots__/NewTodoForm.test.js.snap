// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Passes Snapshot 1`] = `
<DocumentFragment>
  <form>
    <label
      for="task"
    >
      Task
    </label>
    <input
      id="task"
      name="task"
      type="text"
      value=""
    />
    <button>
      Add new Todo!
    </button>
  </form>
</DocumentFragment>
`;
